npm-packages/meteor-babel/options.js

```diff
@@ -85,11 +85,6 @@ exports.getDefaults = function getDefaults(features) {
 function maybeAddReactPlugins(features, options) {
   if (features && features.react) {
     options.presets.push(require("@babel/preset-react"));
-    options.plugins.push(
-      [require("@babel/plugin-proposal-class-properties"), {
-        loose: true
-      }]
-    );
```

Comment:
But `@babel/plugin-proposal-class-properties` was replaced by `@babel/plugin-transform-class-properties`, right? So this should be changed as we did [here](https://github.com/meteor/meteor/pull/13193/files#diff-9b2a6eb1239116bfebc2a178b41aa29996dc1a3f31c8ca24eb66f1f924c7ffdbR89) I believe.